Gruntfile.js

```diff
@@ -13,7 +13,7 @@ module.exports = function(grunt) {
 
     eslint: {
       options: {
-        config: '.eslintrc'
+        config: '.eslintrc.json'
```

Comment:
Tried to find the specific blog post about it, but so far came across only the part here http://eslint.org/docs/user-guide/configuring#configuration-file-formats mentioning that 

> Deprecated - use .eslintrc, which can be either JSON or YAML.